lib/splitclient-rb/clients/split_client.rb

```diff
@@ -62,22 +63,22 @@ def get_treatment(key, split_name, attributes = nil, split_data = nil, store_imp
       end
 
       start = Time.now
-      treatment_with_label = { label: Engine::Models::Label::EXCEPTION, treatment: Treatments::CONTROL }
+      treatment_label_change_number = { label: Engine::Models::Label::EXCEPTION, treatment: Treatments::CONTROL }
```

Comment:
@adilaijaz everywhere we can we just log an exception and continue execution, i.e. if `changeNumber` will be fetched it should be reported back to the Split API